### PR TITLE
Specify engine in Dream API Call

### PR DESCRIPTION
**Pull request by wasabipesto**
Dreamstudio [deprecated some engines](https://platform.stability.ai/docs/features/api-parameters#engine) and so a new one must be specified.

### CODE REVIEW

**Comment by IanPhilips** [comment]
Hmm this doesn't seem to work on the dev preview, we can try prod

**Comment by IanPhilips** [comment]
Seems like it doesn't return an image

Screenshot 2023-11-28 at 10 14 23 AM

**Comment by IanPhilips** [comment]
Let me try again, I think I made a spelling error

**Comment by IanPhilips** [comment]
Thanks @wasabipesto!

**Comment by wasabipesto** [comment]
I'm not sure this is 100% fixed - I changed to the SDXL v0.9 model since that was the first one that worked from my testing, but it seems like the vercel function might be timing out while it waits for the image response. The function times out after 15s but the image generation can take 20-40s.

**Comment by IanPhilips** [comment]
Oh I see, we can fix that if it's a real problem. I haven't experienced it yet

**Comment by wasabipesto** [comment]
Well it's working fine on prod now, maybe dev has different timeout limits.

One last thing, I would keep an eye on usage costs. I don't know how much more the SDXL API costs but there was a note that it would be higher than the old engine.
